perma_web/perma/celery_tasks.py

```diff
@@ -1224,3 +1229,104 @@ def conditionally_queue_internet_archive_uploads_for_date_range(start_date_strin
 
     else:
         logger.info("Skipped the queuing of file upload tasks: max tasks already in progress.")
+
+
+# WACZ CONVERSION
+
+def format_time(seconds_val):
```

Comment:
The tiniest of suggestions: since this is such a context-specific utility, what would you think of defining it inside the scope of `convert_warc_to_wacz`? That is a pattern I personally enjoy and find tidy when I have a helper with a very narrow use (for example, [here](https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/celery_tasks.py#L535), but some people like it less. If you don't prefer that approach, how about naming this function "seconds_to_minutes"?